charts/tempo-distributed/README.md

```diff
@@ -1,6 +1,6 @@
 # tempo-distributed
 
-![Version: 1.4.13](https://img.shields.io/badge/Version-1.4.13-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 2.1.1](https://img.shields.io/badge/AppVersion-2.1.1-informational?style=flat-square)
+![Version: 1.5.0](https://img.shields.io/badge/Version-1.5.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 2.2.0-rc.0](https://img.shields.io/badge/AppVersion-2.2.0--rc.0-informational?style=flat-square)
```

Comment:
I only bumped this chart a minor release because I noticed that's the pattern we've used previously when removing outdated configs, for example: https://github.com/grafana/helm-charts/commit/0590650f979ccb68f53c735addc11ee1b383a669

Reply:
1.5.x seems okay to me for the major version, and the backwards incompatibilities.  Drawing folks attention to the changelog/readme would be good, and not sure folks look all the time for minor version changes.

Reply:
I added this section to the readme, which links to the relevant portions of the Tempo changelog:

> ## From Chart versions < 1.5.0
> 
> Please be aware that we've updated the minor version to Tempo 2.2, which includes breaking changes.
> We recommend reviewing the [release notes](https://github.com/grafana/tempo/releases/tag/v2.2.0-rc.0/) before upgrading.

Lmk if there's something else I should add or change :+1: